src/module/actor/character/sheet.ts

```diff
@@ -79,6 +79,12 @@ class CharacterSheetPF2e<TActor extends CharacterPF2e> extends CreatureSheetPF2e
     /** Non-persisted tweaks to formula data */
     #formulaQuantities: Record<string, number> = {};
 
+    /** PC Sheet Tab manager instance. Must be destroyed to avoid memory leaks  */
+    #tabManager: PCSheetTabManager | null = null;
+
+    /** Elements with registered $.tooltipster instances. Have to be cleaned up to avoid memory leaks */
+    #tooltipsterElements: JQuery[] = [];
```

Comment:
Any reason to not handle this from the base class?

Reply:
No reason at all, should be better now